src/formatter.js

```diff
@@ -26,7 +27,7 @@ assign(Formatter.prototype, {
   // a "joining" value is specified (e.g. ' and ')
   parameterize(values, notSetValue) {
     if (typeof values === 'function') return this.parameter(values);
-    values  = Array.isArray(values) ? values : [values];
+    values  = isArray(values) ? values : [values];
```

Comment:
Not sure I follow on this change. `const { isArray } = Array;` aren't you then constructing a new Array with values here? Same as `Array(null)` `Array(void 0)` `Array([1,2,3])`. Would mean the ternary operation **cant** fail?

Or am I just missing something obvious?